src/scaffoldmaker/meshtypes/meshtype_3d_lung2.py

```diff
@@ -0,0 +1,2139 @@
+            options['Right lung medial protrusion factor'] = 0.7
+            options['Accessory lobe'] = False
+            options['Diaphragm curvature x'] = 1.0
+            options['Diaphragm curvature y'] = 1.0
+            options['Rotation in degree about [z, y, x]: Left lung'] = [25.0, 0.0, 0.0]
+            options['Rotation in degree about [z, y, x]: Right lung'] = [-25.0, 0.0, 0.0]
+            options['Rotation in degree about [z, y, x]: Accessory lobe'] = [0.0, 0.0, 0.0]
+        elif 'Mouse 1' in parameterSetName:
+            options['Number of left lung lobes'] = 1
+            options['Left-right lung spacing'] = 0.75
+            options['Left-right apex closing space'] = 0.7
+            options['Left-right apex along y-axis'] = -1.3
+            options['Left lung width'] = 0.25
+            options['Left lung depth'] = 0.5
+            options['Left lung height'] = 0.7
+            options['Left lung ventral edge sharpness factor'] = 1.0
+            options['Left lung medial curvature factor'] = 1.5
+            options['Right lung width'] = 0.4
+            options['Right lung depth'] = 0.55
+            options['Right lung height'] = 0.85
+            options['Right lung ventral edge sharpness factor'] = 1.0
+            options['Right lung medial curvature factor'] = 2.0
+            options['Accessory lobe dorsal centre [x,y]'] = [0.0, 0.2]
+            options['Accessory lobe length'] = 0.5
+            options['Accessory lobe dorsal height'] = 0.5
+            options['Accessory lobe dorsal width'] = 0.5
+            options['Accessory lobe ventral height'] = 0.15
+            options['Accessory lobe ventral width'] = 0.28
+            options['Accessory lobe medial curvature'] = 2.0
+            options['Diaphragm centre y'] = 0.5
+            options['Diaphragm curvature x'] = 0.8
+            options['Diaphragm curvature y'] = 1.0
+            options['Rotation in degree about [z, y, x]: Left lung'] = [-20.0, 0.0, 0.0]
+            options['Rotation in degree about [z, y, x]: Right lung'] = [-10.0, 0.0, 0.0]
+            options['Rotation in degree about [z, y, x]: Accessory lobe'] = [-80.0, 0.0, 0.0]
+        elif 'Rat 1' in parameterSetName:
+            options['Number of left lung lobes'] = 1
+            options['Left-right lung spacing'] = 0.9
+            options['Left-right apex closing space'] = 1.3
+            options['Left-right apex along y-axis'] = 0.8
+            options['Left lung width'] = 0.25
+            options['Left lung depth'] = 0.7
+            options['Left lung height'] = 1.0
+            options['Left lung ventral edge sharpness factor'] = 0.6
+            options['Left lung medial curvature factor'] = 1.0
+            options['Left lung medial protrusion factor'] = 0.8
+            options['Right lung width'] = 0.18
+            options['Right lung depth'] = 0.9
+            options['Right lung height'] = 1.0
+            options['Right lung ventral edge sharpness factor'] = 0.8
```

Comment:
The Right and (to a lesser extent Left) sharpness is too great on the rat -- I think you should assume the rat data is quite deflated so choose options to inflate it a bit, so more similar to the state of the other species, particularly the mouse which I'd think it would be similar to.
I notice the pig has even higher sharpness factors; I'm not sure what other options cause it to not look as sharp as the rat here?

Reply:
I fixed the parameters for the rat. This is because of the width of the right lung which causes the right lung is even more sensitive to its sharpness parameter